.github/workflows/linux-build-and-test.yaml

```diff
@@ -66,7 +66,11 @@ jobs:
     - name: Build PostgreSQL ${{ matrix.pg }} ${{ matrix.build_type }}
```

Comment:
Shouldn't this be `Build PostgreSQL ${{ matrix.pg }} ${{matrix.snapshot}} ${{ matrix.build_type }}`?

And more importantly, shouldn't the cache above it include the `snapshot` otherwise it will conflict on the same key?

Reply:
The key won't conflict since matrix.pg is either 12 or 13 for snapshots while it's exact version for non-snapshots e.g. 13.4
But i should probably disable cache entirely for snapshots.